2024-06-06 00:40:11.976 [info] 




2024-06-06 00:40:11.976 [info] Extension host agent started.
2024-06-06 00:40:11.976 [info] Started initializing default profile extensions in extensions installation folder. file:///home/gustavo/.vscode-server/extensions
2024-06-06 00:40:12.015 [info] [<unknown>][0eea4e1f][ManagementConnection] New connection established.
2024-06-06 00:40:12.017 [info] [<unknown>][e22471f5][ExtensionHostConnection] New connection established.
2024-06-06 00:40:12.183 [info] [<unknown>][e22471f5][ExtensionHostConnection] <1289> Launched Extension Host Process.
2024-06-06 00:40:13.094 [info] Completed initializing default profile extensions in extensions installation folder. file:///home/gustavo/.vscode-server/extensions
2024-06-06 00:40:29.656 [info] [<unknown>][728c7e23][ExtensionHostConnection] New connection established.
2024-06-06 00:40:29.660 [info] [<unknown>][728c7e23][ExtensionHostConnection] <1503> Launched Extension Host Process.
2024-06-06 00:40:29.661 [info] [<unknown>][73ffe339][ManagementConnection] New connection established.
2024-06-06 00:40:32.045 [error] CodeExpectedError: Could not find pty on pty host
    at b.U (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:32991)
    at b.updateTitle (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:28152)
    at A.<computed> (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:23406)
    at Object.call (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5112)
    at a.s (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5118)
    at a.q (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4636)
    at n.value (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at i.y (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:670)
    at i.fire (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:889)
    at process.$e (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10159)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-06-06 00:45:11.921 [info] New EH opened, aborting shutdown
2024-06-06 00:45:11.922 [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/server/node/server.main.js:189:6057)
    at process.emit (node:events:517:28)
2024-06-06 00:45:29.911 [info] [<unknown>][e22471f5][ExtensionHostConnection] <1289> Extension Host Process exited with code: 0, signal: null.
2024-06-06 01:53:03.618 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:55:08.672 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:55:08.672 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:55:08.691 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:55:08.692 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:55:08.912 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 01:58:00.513 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.150 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.913 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.915 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.916 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.917 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:12:38.918 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:18:43.153 [warning] No ptyHost heartbeat after 6 seconds
2024-06-06 02:20:55.215 [warning] No ptyHost heartbeat after 6 seconds
2024-06-06 02:20:57.684 [info] [<unknown>][728c7e23][ExtensionHostConnection] The client has reconnected.
2024-06-06 02:29:38.100 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:30:03.497 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:30:03.748 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
2024-06-06 02:31:56.301 [error] CodeExpectedError: Could not find pty on pty host
    at b.U (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:32991)
    at b.updateIcon (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:28203)
    at A.<computed> (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:23406)
    at Object.call (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5112)
    at a.s (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5118)
    at a.q (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4636)
    at n.value (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at i.y (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:670)
    at i.fire (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:889)
    at process.$e (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10159)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-06-06 02:31:56.447 [error] CodeExpectedError: Could not find pty on pty host
    at b.U (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:32991)
    at b.updateTitle (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:28152)
    at A.<computed> (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:23406)
    at Object.call (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5112)
    at a.s (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5118)
    at a.q (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4636)
    at n.value (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at i.y (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:670)
    at i.fire (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:889)
    at process.$e (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10159)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-06-06 02:31:56.448 [error] CodeExpectedError: Could not find pty on pty host
    at b.U (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:32991)
    at b.updateTitle (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:28152)
    at A.<computed> (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:52:23406)
    at Object.call (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5112)
    at a.s (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5118)
    at a.q (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4636)
    at n.value (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at i.y (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:670)
    at i.fire (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:29:889)
    at process.$e (/home/gustavo/.vscode-server/cli/servers/Stable-dc96b837cf6bb4af9cd736aa3af08cf8279f7685/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10159)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-06-06 02:39:26.821 [error] AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at afterConnectMultiple (node:net:1601:5)
